azure/packages/test/end-to-end-tests/src/test/audience.spec.ts

```diff
@@ -161,4 +156,172 @@ describe("Fluid audience", () => {
 		members = servicesGet.audience.getMembers();
 		assert.strictEqual(members.size, 1, "We should have one member left at this point.");
 	});
+
+	/**
+	 * Scenario: Find read-only partner member
+	 *
+	 * Expected behavior: upon resolving container, the read-only partner member should be able
+	 * to resolve original member, and the original member should be able to observe the read-only member.
+	 */
+	it("can find read-only partner member", async function () {
+		if (process.env.FLUID_CLIENT !== "azure") {
+			// Tinylicious does not support read-only mode
```

Comment:
Do you know if we plan on having tinylicious support read-only?

Reply:
I have no idea. Also, seeing your comment from earlier, not entirely sure how read-only was working or why it made these fail... Gonna check on that.

Reply:
Looks like these failures were because read-only tests don't work in _either_ Azure or T9s, but I was only seeing it in T9s because I forgot to pass `scopes` all the way in Azure.